sources/docker-compose.yml

```diff
@@ -68,6 +68,7 @@ services:
     depends_on:
       - postgres
       - redis
+      - clamav
```

Comment:
I believe we should review the AC.
ClamAV will not be a dependency for the API to be executed.
Maybe it should not be a dependency at all since even only one `queue-consumers` queue will depend on it.
We may add it as a dependency for `queue-consumers`, but not mandatory IMO.